src/whatsnew.adoc

```diff
@@ -95,95 +102,89 @@ their position in the table (i.e., the `$id` syntax).
-with inline text.  This encoding is structurally identical to `VarInt` but is used to encode symbolic tokens that could
-be either a symbol ID or a length of the UTF-8 octets that follow.
-
-* Positive integers represent symbol IDs.
-
-* Negative integers represent a length of UTF-8 octets to follow.
-
-* Zero is treated specially.  Specifically, A `VarSym` encoded as `0x00` indicates that a special control opcode
-follows. The following table specifies the control opcodes, all other opcodes are illegal.
-
-[%header,%unbreakable,cols="1,1"]
-|===
-
-| Control Opcode
-| Description
-
-| `0x0_` - `0x4_`, `0xE1`, `0xF0`
-| E-expression follows.  This is only valid when a field name in a struct is expected.
-
-| `0x70`
-| Represents `$0`.
-
-| `0x80`
-| Represents empty inline text (i.e., `''`).
-
-| `0xAD`
-| Ends a delimited struct.  This is only valid when a field name in a _delimited_ struct is expected.
-
-|===
-
-The padded representations of zero in `VarSym` (e.g., `0x80 0x00`, `0xC0 0x00 0x00`) all mean `$0`.
-
-NOTE: The _control opcode_ is designed to parallel the Ion 1.1 _type opcodes_ which is why they might seem, out of that
-context, arbitrary.
+containers to be encoded with an end marker instead of a length prefix.
+
+==== Low-level Binary Encoding Changes
+
+Ion 1.0's link:https://amazon-ion.github.io/ion-docs/docs/binary.html#varuint-and-varint-fields[`VarUInt` and `VarInt`
+ encoding primitives] used big-endian byte order and used the high bit of each byte to indicate whether it was the final
+byte in the encoding.
+`VarInt` used an additional bit in the first byte to represent the integer's sign.
+Ion 1.1 replaces these primitives with more optimized versions called
+link:binary-encoding.adoc#flexuint[`FlexUInt`] and link:binary-encoding.adoc#flexint[`FlexInt`].
+
+link:binary-encoding.adoc#flexuint[`FlexUInt`] and link:binary-encoding.adoc#flexint[`FlexInt`] use little-endian byte
+order, avoiding the need for reordering on x86 architectures.
+Rather than using a bit in each byte to indicate the width of the encoding, `FlexUInt` and `FlexInt` front-load the
+continuation bits.
+In most cases, this means that these bits all fit in the first byte of the representation, allowing a
+reader to determine the complete size of the encoding without having to inspect each byte individually.
+Finally, `FlexInt` does not use a separate bit to indicate its value's sign. Instead, it uses two's complement
+representation, allowing it to share much of the same structure and parsing logic as its unsigned counterpart.
+Benchmarks have shown that in aggregate, these encoding changes are between 1.25 and 3x faster than Ion 1.0's
+`VarUInt` and `VarInt` encodings depending on the host architecture.
+
+Ion 1.1 supplants link:https://amazon-ion.github.io/ion-docs/docs/binary.html#uint-and-int-fields[Ion 1.0's `Int`
+encoding primitive] with a new encoding called link:binary-encoding.adoc#fixedint[`FixedInt`], which uses two's
```

Comment:
Have we also renamed `UInt` to `FixedUInt`, and should we mention it here?

Reply:
> Have we also renamed `UInt` to `FixedUInt`

It has, yes.

> and should we mention it here?

I only omitted it because the encoding is the same as `UInt` so there wasn't much to say. I've added a sentence making that explicit.